### DISCUSSION TITLE
Github page directing to custom domain

### DISCUSSION THREAD

**Discussion by maxence33** [community/community#23821] [Other Feature Feedback, Questions, & Ideas]
<p>I had troubles creating a custom domain properly. It first worked and then went bananas.<br>
Then I have reset my custom domain DNS in order to start from blank page.<br>
I also have removed the custom domain from my Github repo page.</p>
<p>Then Github says I can access again my GH page at the usual link : <a href="http://my-account.github.io/my-repo" rel="nofollow noopener">my-account.github.io/my-repo</a></p>
<p>Though when I click it I am redirected to the custom domain I have mentioned. Is there any reason for this ?</p>
<p>Thanks you</p>

**Comment by lee-dohm** [comment]
<p>Thanks for joining us :slight_smile:</p>
<p>We don’t have access to private account details here, but have you checked <a href="https://help.github.com/en/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain">your repository’s custom domain configuration</a> to ensure that you removed the reference?</p>
<p>When I request your site, I get a redirect from the GitHub Pages server:</p>
<pre><code class="lang-auto">$ http https://maxence33.github.io/narrative-video-player/

HTTP/1.1 301 Moved Permanently
Accept-Ranges: bytes
Age: 185
Connection: keep-alive
Content-Length: 162
Content-Type: text/html
Date: Thu, 28 May 2020 18:20:34 GMT
Location: http://narrativeplayer.com/
Server: GitHub.com
Vary: Accept-Encoding
Via: 1.1 varnish
X-Cache: HIT
X-Cache-Hits: 1
X-Fastly-Request-ID: 2d1455a6b800b4db0cfb58b8167de101159ea697
X-GitHub-Request-Id: 21CE:46F5:DF591:104B56:5ED00038
X-Served-By: cache-sea4451-SEA
X-Timer: S1590690034.142668,VS0,VE0

&lt;html&gt;
&lt;head&gt;&lt;title&gt;301 Moved Permanently&lt;/title&gt;&lt;/head&gt;
&lt;body&gt;
&lt;center&gt;&lt;h1&gt;301 Moved Permanently&lt;/h1&gt;&lt;/center&gt;
&lt;hr&gt;&lt;center&gt;nginx&lt;/center&gt;
&lt;/body&gt;
&lt;/html&gt;
</code></pre>

**Comment by maxence33** [comment]
<p>Hi,<br>
Thanks for your answer. Yes I removed the reference in settings/custom domain.<br>
But I was forwarded to narrativeplayer com each time… Couldn’t access original GH pages when the custom domain was removed from GH and the DNS of narrativeplayer wiped.</p>
<p>Though I have reinstated it as I tried a different approach.<br>
Originally I made an A record internally at OVH from my root to www. And a CNAME from my www to <a href="http://maxence33.github.io" rel="nofollow noopener">maxence33.github.io</a><br>
It worked for some time but then broke.</p>
<p>I am now pointing the Apex directly to Github hard IPs<br>
then when I dig it : dig <a href="http://narrativeplayer.com" rel="nofollow noopener">narrativeplayer.com</a> I can now see the two records :<br>
|narrativeplayer com.|3294|IN|A|185.199.108.153|<br>
|narrativeplayer com.|3294|IN|A|185.199.110.153|</p>
<p>Though it still doesn’t work.<br>
I think I will wait until tomorrow to investigate further, because of propagation times …</p>

**Comment by lee-dohm** [comment, marked as answer]
<aside class="quote no-group" data-username="maxence33" data-post="3" data-topic="115768">
<div class="title">
<div class="quote-controls"></div>
 maxence33:</div>
<blockquote>
<p>Though it still doesn’t work.</p>
</blockquote>
</aside>
<p>I’m not entirely certain what it is you’re saying isn’t working. Here’s what I see when I go to <a href="http://narrativeplayer.com">http://narrativeplayer.com</a>:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/4c381078c78df0ca6ecdee3f4b938178d6f43a78.png" data-download-href="/uploads/short-url/aSgp71VCc7Hasr7MgGTKfY9E7Es.png?dl=1" title="Screen Shot 2020-05-28 at 12.00.12 PM"><img src="https://user-images.githubusercontent.com/1038121/181079429-8ba962dd-c28f-4f36-83ec-7a6af4afa6ae.png" alt="Screen Shot 2020-05-28 at 12.00.12 PM" data-base62-sha1="aSgp71VCc7Hasr7MgGTKfY9E7Es" width="517" height="338" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/4c381078c78df0ca6ecdee3f4b938178d6f43a78_2_517x338.png, https://aws1.discourse-cdn.com/github/optimized/2X/4/4c381078c78df0ca6ecdee3f4b938178d6f43a78_2_775x507.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/4/4c381078c78df0ca6ecdee3f4b938178d6f43a78_2_1034x676.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/4c381078c78df0ca6ecdee3f4b938178d6f43a78_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2020-05-28 at 12.00.12 PM</span><span class="informations">1544×1010 102 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Can you be more specific and share:</p>
<ol>
<li>Step-by-step what you’re doing</li>
<li>What you expect to happen</li>
<li>What actually happens instead?</li>
</ol>

**Comment by maxence33** [comment]
<p>Oh fantastic !<br>
Thank you.<br>
In Firefox I was typing the address but not sure why it added subdomain www.<br>
I banged my head as I made no CNAME between apex and www in my last attempt so I thought it was a glitch or a DNS propagation issue.<br>
I had to click on the little shield just at the left of the firefox navbar and kill “cookies”.<br>
Now I can see it !</p>
<p>Chrome still gives me DNS_PROBE_FINISHED_NXDOMAIN but it won’t let me remove the subdomain www when I type address. I will try to find where the problem is with Chrome.<br>
Thanks a lot !!</p>
